code/index.html

```diff
@@ -1,22 +1,134 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
 
-    <section class="">
-      <!-- The other news  -->
+<head>
+  <meta charset="utf-8" />
+  <title>Todays Inspiration</title>
+  <link rel="icon" href="./only-cat.logo.svg" />
+  <link rel="stylesheet" href="./style.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300;700&family=Open+Sans:wght@300;400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+<!-- Header with logo and nav -->
+  <header>
+    <img class="logoIcon" src="./cat.logo2.svg">
```

Comment:
Remember to use the alt om images to further explain what we're looking at, here's some info regarding this but it's nice to know: 
https://www.w3schools.com/tags/att_img_alt.asp